### PR TITLE
Grpc stream push

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: oomagent/test/data/user_click.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description: 'user id'
+groups:
+  - name: user-click
+    category: stream
+    description: user click post feature
+    features:
+      - name: last_5_click_posts
+        value-type: string
+        description: user last 5 click posts
+      - name: number_of_user_started_posts
```
Comment (YikSanChan) [inline oomagent/test/data/user_click.yaml:12]: typo: started -> starred

File: oomagent/test/data/user_click.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        value-type: int64
+        description: number of posts that users stared today
```
Comment (YikSanChan) [inline oomagent/test/data/user_click.yaml:14]: stared -> starred

**Review by YikSanChan — APPROVED** [review]
lgtm

File: oomagent/test/data/user_click.yaml (hunk 1 above)
Comment (lianxmfor) [reply oomagent/test/data/user_click.yaml]: fix

**Pull request merged**